src/journal-remote/journal-upload.c

```diff
@@ -279,6 +408,10 @@ int start_upload(Uploader *u,
                 u->answer = mfree(u->answer);
         }
 
+        r = refresh_timeout(u);
+        if (r < 0)
+                return log_error_errno(r, "Failed to refresh uploader timer");
```

Comment:
duplicate logging. refresh_timeout() already logs about all errors afaics

Reply:
replaced it with a plain code return